### PR TITLE
ADEN-13571 | don't allow prebid.js and reorg pipeline for it

### PR DESCRIPTION
**Pull request by mfarul**

### CODE REVIEW

**Review by jakubbanas — DISMISSED** [review]

File: src/platforms/shared/tracking/metric-reporter.ts [hunk 1]
```diff
@@ -30,12 +37,13 @@ export class MetricReporter {
 		this.isActive = utils.outboundTrafficRestrict.isOutboundTrafficAllowed('monitoring-default');
 	}
 
-	execute() {
+	initialise() {
```
Comment (preludek) [inline src/platforms/shared/tracking/metric-reporter.ts:40]:
```suggestion
	initialize() {
```

😈

File: src/platforms/shared/utils/preloaded-libraries-setup.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+		if (this.instantConfig.get('icA9Bidder')) {
+			A9Provider.initApstag();
+		}
+
+		if (this.instantConfig.get('icAudigent')) {
+			Audigent.loadSegmentLibrary();
+		}
+	}
+
+	pu;
```
Comment (preludek) [inline src/platforms/shared/utils/preloaded-libraries-setup.ts:47]: coś zostało 😅 
image

**Review by preludek — CHANGES_REQUESTED** [review]
Left one comment and there's also a random line to be removed.

File: src/platforms/shared/utils/preloaded-libraries-setup.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+import { Injectable } from '@wikia/dependency-injection';
+import { GptSetup } from '../setup/gpt.setup';
+
+const prebidLibraryUrlLocation = '//static.wikia.nocookie.net/fandom-ae-assets/prebid.js/';
+const prebidLibraryUrlPattern = new RegExp(
+	`^(${prebidLibraryUrlLocation.replaceAll('.', '\\.')})?[^/]+/[^/]+$`,
+);
+
+@Injectable()
+export class PreloadedLibrariesSetup implements DiProcess {
```
[the remark this answers is not in the document]
Comment (mfarul) [reply src/platforms/shared/utils/preloaded-libraries-setup.ts:27]: Yea, folder could be changed, it was just used as i've renamed existing file ;).

About the I've tried but then we need to change the pipeline logic as it's working on DIProcess that does not allow different instances :(.

**Review by preludek — DISMISSED** [review]

**Review by preludek — APPROVED** [review]

**Review by preludek — APPROVED** [review]

**Pull request merged**